websockets/Streaming.WebSocket.Samples/WebSocketSample.cs

```diff
@@ -196,13 +196,20 @@ private async Task ReauthorizeWhenNeeded(DateTime tokenExpiryTime, CancellationT
             }
         }
 
+        private HttpClient CreateHttpClient()
```

Comment:
Can you add a summary above this function?

Reply:
Summary added.